### PR TITLE
docs: update readme for validate commit meesage and format code on save

### PR DESCRIPTION
**Pull request by QaseemLodhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,48 @@ The scope of the application is very simple. We are looking to create a simple e
...
+git commit -m "foo: this will fail"
+husky > npm run -s commitmsg
+
+⧗   input: foo: this will fail
+✖   type must be one of [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test] [type-enum]
+✖   found 1 problems, 0 warnings
+
+husky > commit-msg hook failed (add --no-verify to bypass)
+
+git commit -m "chore: lint on commitmsg"
```
[the remark this answers is not in the document]
Comment (QaseemLodhi) [reply README.md]: like we are doing in bt-api?

[the remark this answers is not in the document]
Comment (mashhoodr) [reply README.md]: yes similar to that

**Comment by mashhoodr** [comment]
@QaseemLodhi please fix and rebase today

**Comment by mashhoodr** [comment]
@QaseemLodhi looks like you didnt merge something properly, the PR should only show your changes. Please create a new PR

**Pull request closed without merging**